packages/reports/src/Components/Routes/binary-routes.jsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import { Switch } from 'react-router-dom';
+import { PlatformContext } from '@deriv/shared';
+import { Localize } from '@deriv/translations';
+import getRoutesConfig from '../../Constants/routes-config';
+import RouteWithSubRoutes from './route-with-sub-routes.jsx';
+
+const BinaryRoutes = props => {
+    const { is_dashboard } = React.useContext(PlatformContext);
```

Comment:
Dashboard has been removed already, no need to have this variable anymore.
Needs to be removed from CFD package as well.

Reply:
sure, will remove it :pray: